static/assets/matrix-introspection.js

```diff
@@ -0,0 +1,132 @@
+// parse the query string
```

Comment:
this is an absolutely straight lift-and-shift from index.html, except for the indentation.